pkg/middleware/middleware.go

```diff
@@ -21,8 +21,9 @@ var (
 	ReqOrgAdmin            = RoleAuth(org.RoleAdmin)
 )
 
-func HandleNoCacheHeader(ctx *contextmodel.ReqContext) {
-	ctx.SkipCache = ctx.Req.Header.Get("X-Grafana-NoCache") == "true"
+func HandleNoCacheHeaders(ctx *contextmodel.ReqContext) {
```

Comment:
It seemed fitting to stick these together, but reading the header in the REST handlers would be an option too

Reply:
Shouldn't your comments in here actually be comments in the code? :P